bnb.rb

```diff
@@ -22,4 +25,15 @@ class BnB < Sinatra::Base
     result = Space.create(name: params['name'])
     "#{result.name} has been listed"
   end
+
+  get '/users/new' do
+    erb :'users/new'
+  end
+
+  post '/users' do    
+    user = User.create(first_name: params['firstname'], last_name: params['lastname'], password: params['password'], email: params['email'])
```

Comment:
Would it be worth formatting this to reduce the line length? (each argument on its own line to improve readability)

Reply:
Sounds good, we will change this